src/app/containers/Header/a11y.test.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { service as newsServiceConfig } from '#lib/config/services/news';
+import { FRONT_PAGE } from '#app/routes/utils/pageTypes';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { ToggleContext } from '#contexts/ToggleContext';
+import HeaderContainer from './index';
+
+const bannerMock = require('../ConsentBanner');
+const brandMock = require('../Brand');
```

Comment:
Am I right in thinking we are mocking out the components we want to test and then reimplementing them below?